oss/node-8/up.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "app",
+  "hooks": {
+    "build": "docker run --rm -w /tmp -v $(PWD):/tmp romuloaalves/up-node8 pkg --output=server --targets node8-linux .",
+    "clean": "rm server"
```

Comment:
first of all thanks! this seems not bad at all, still builds pretty quick. Stupid question, I don't know exactly what `pkg` does, but don't we only need a node8 binary? Could we just grab that from https://nodejs.org/en/download/ and potentially avoid docker? I'll give it a try in a minute